### PR TITLE
VerticalResults: add appliedFilters config to readme

### PR DESCRIPTION
**Pull request by oshi97**
showChangeFilters is only applicable to UniversalResults, so was
not listed

v1.4.0 QA: 12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,6 +565,22 @@ ANSWERS.addComponent('VerticalResults', {
...
+  },
+  // Configuration for the applied filters bar in the header.
+  appliedFilters: {
+    // If true, show any applied filters that were applied to the universal search. Defaults to false
+    show: true,
+    // If appliedFilters.show is true, whether to display the field name of an applied filter, e.g. "Location: Virginia" vs just "Virginia". Defaults to false.
+    showFieldNames: false,
+    // If appliedFilters.show is true, this is list of filters that should not be displayed.
+    // By default, builtin.entityType will be hidden
+    hiddenFields: ['builtin.location'],
```
Comment (tmeyer2115) [inline README.md]: Update

File: README.md [hunk 2]
```diff
@@ -565,6 +565,22 @@ ANSWERS.addComponent('VerticalResults', {
...
+    // The character that separates the count of results (e.g. “1-6”) from the applied filter bar. Defaults to '|'
+    resultsCountSeparator: '|',
+    // If the filters are shown, whether or not they should be removable from within the applied filter bar. Defaults to false.
+    removable: false,
+    // The character that separates each field (and its associated filters) within the applied filter bar. Defaults to '|'
+    delimiter: '|'
```
Comment (tmeyer2115) [inline README.md:583]: Do we need to add labelText and removableLabelText as well?

Comment (oshi97) [reply README.md:583]: adding that in a separate pr!

File: README.md (hunk 1 above)
Comment (oshi97) [reply README.md]: merged pr into this branch

File: src/ui/components/results/verticalresultscomponent.js [hunk 3]
```diff
@@ -92,7 +92,7 @@ class VerticalResultsConfig {
        * Any fieldIds in hiddenFields will be hidden from the list of appied filters.
        * @type {Array<string>}
        */
-      hiddenFields: defaultConfigOption(config, ['appliedFilters.hiddenFields', 'hiddenFields'], ['builtin.location']),
```
Comment (tmeyer2115) [inline src/ui/components/results/verticalresultscomponent.js]: This used to be part of another PR. Was there a rebasing issue?

Comment (oshi97) [reply src/ui/components/results/verticalresultscomponent.js]: Ah I noticed github automatically rebases relation changes, so I wanted to try something like setting this PR as the base for that second PR. I guess that only works if this one gets merged in first, though. I'll rebase!

**Review by tmeyer2115 — APPROVED** [review]
